admin/admin-filters-sanitize.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Setup specific admin filters useful for sanitization.
+ *
+ * Extract from PLL_Admin_Filters to be able to use in a REST API context.
+ *
+ * @since 2.9
+ */
+class PLL_Admin_Filters_Sanitize {
+	/**
+	 * Current language used for the sanitization depending on the context.
+	 *
+	 * @var string $curlang
+	 */
+	public $curlang;
+
+	/**
+	 * Instance of PLL_Model
+	 *
+	 * @var PLL_Model
+	 */
+	protected $model;
+	
+	/**
+	 * Constructor: setups filters and actions
+	 *
+	 * @since 2.9
+	 *
+	 * @param object $polylang
+	 */
+	public function __construct( &$polylang ) {
+		$this->model = &$polylang->model;
+		$this->curlang = &$polylang->curlang;
```

Comment:
`$curlang` property doesn't exists in `PLL_REST_Request`. By assigning a shared reference here, you are dynamically creating it in the `$polylang` object.

Reply:
$curlang has been replaced by $locale property which is used only internally to PLL_Filters_Sanitization class